package.json

```diff
@@ -4,6 +4,11 @@
   "description": "An item manager for Destiny.",
   "main": "app/index.html",
   "scripts": {
+    "dev-chrome": "./node_modules/.bin/grunt dev-chrome",
+    "dev-firefox": "./node_modules/.bin/grunt dev-firefox",
+    "generate-css": "./node_modules/.bin/grunt css",
```

Comment:
In the examples it's `npm run css`, but should it be `npm run generate-css`?

Will we have to mirror every Grunt task into this?